docs/reference/batch-processing/api-torch-batch-process-reference.rst

```diff
@@ -0,0 +1,51 @@
+:orphan:
+
+.. _torch-batch-process-api-ref:
+
+#######################################
+ ``Torch Batch Process`` API Reference
+#######################################
+
```

Comment:
something about the title does not look consistent with the other API ref page titles

should the title be more like this?

#######################################
 ``pytorch.torch_batch_process`` API Reference
#######################################